sdks/python/apache_beam/testing/test_pipeline.py

```diff
@@ -146,6 +144,7 @@ def _parse_test_option_args(self, argv):
     known, unused_argv = parser.parse_known_args(argv)
 
     if self.is_integration_test and not known.test_pipeline_options:
+      from nose.plugins.skip import SkipTest
```

Comment:
Agree to limit usage of nose for future pytest migration. I think the usage here is the same as [`TestCase.skipTest`](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.skipTest), can we directly use the this standard library?

Reply:
Done